docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+
+### Browsers in Cloud Testing Services
+
+TestCafe allows you to use browsers from cloud testing services. You can access them through [browser provider plugins](../extending-testcafe/browser-provider-plugin/README.md).
+
+The following plugins for cloud services are currently provided by the TestCafe team.
+
+Service                              | Plugin
+------------------------------------ | -------------------
+[Sauce Labs](https://saucelabs.com/)          | [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs)
+[BrowserStack](https://www.browserstack.com/) | [testcafe-browser-provider-browserstack](https://www.npmjs.com/package/testcafe-browser-provider-browserstack)
+
+You can search npm for plugins developed by the community. Their names begin with the `testcafe-browser-provider-` prefix: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
+
+You can also create your own plugin. See [Browser Provider Plugin](../extending-testcafe/browser-provider-plugin/README.md) for instructions.
+
+### Microsoft Edge Legacy Support
+
+Tests are run in Microsoft Edge Legacy if it is the only version of Edge installed on the machine.
+
+If Chromium-based and Legacy versions are available, you can change the default Edge application in **System Settings** to test in the Legacy version.
+
+To do this, open **Default apps** in **Windows Settings** and scroll down to select the option **Default Apps by Protocol**. Set the `MICROSOFT-EDGE:` protocol to Legacy Edge.
+
+### Nonconventional Browsers
+
+To use a different type of browser, use a [browser provider plugin](../extending-testcafe/browser-provider-plugin/README.md).
+
+You can search npm for plugins developed by the community. Their names begin with the `testcafe-browser-provider-` prefix: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
+
+You can also create your own plugin. See [Browser Provider Plugin](../extending-testcafe/browser-provider-plugin/README.md) for instructions.
+
+## Test in Headless Mode
+
+TestCafe allows you to run tests in Google Chrome and Mozilla Firefox without any visible UI shell - in the headless mode ([Chrome Headless](https://developers.google.com/web/updates/2017/04/headless-chrome), [Firefox Headless](https://developer.mozilla.org/en-US/Firefox/Headless_mode)). Use the `:headless` parameter to launch a browser in the headless mode.
+
+```sh
+testcafe "chrome:headless" tests/sample-fixture.js
+```
+
+```js
+runner
+    .src('tests/sample-fixture.js')
+    .browsers('chrome:headless')
+    .run();
+```
+
+Specify a path to the browser executable if you use a portable version of the browser. Use the [browser alias](locally-installed-browsers) instead of the `path:` prefix.
```

Comment:
What do you mean by "portable version"? Do you mean a mobile browser?

Reply:
No, portable are desktop browsers that do not require installation.

https://en.wikipedia.org/wiki/Portable_application

Reply:
Okay